lib/request.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const request = require('request')
+const Backoff = require('backo')
```

Comment:
Do we need to add backo for this? I think its sensible to avoid too many deps and this one seems very simple. We can just wait 1000 each time for example?

Reply:
> Do we need to add _X_ for this? I think its sensible to avoid too many deps

You're speaking my language! Here's my justification:

`backo` is my go-to dep for this – I've used it successfully in high-traffic situations, it's simple, does one thing well, and has no dependencies of it's own (which means for us there are no transitive dependencies – an important point). I'd rather add 10 "flat" dependencies than 1 "iceberg".

> We can just wait 1000 each time for example?

We can do that. I think the best approach for this kind of thing in general is an exponential backoff, but it would be trival to remove `backo` and use a static timeout value.

Let me know what approach you want me to take.